packages/ui/src/theme/css/component/inAppMessaging/banner.scss

```diff
@@ -0,0 +1,60 @@
+.amplify-in-app-messaging-banner__button {
+  background-color: var(--amplify-components-inappmessagingbanner-button-background-color);
+  border-style: var(--amplify-components-inappmessagingbanner-button-border-style);
+  width: var(--amplify-components-inappmessagingbanner-button-width);
+}
+.amplify-in-app-messaging-banner__container {
+  background-color: var(--amplify-components-inappmessagingbanner-banner-container-background-color);
+  box-shadow: var(--amplify-components-inappmessagingbanner-banner-container-box-shadow);
+  flex-direction: var(--amplify-components-inappmessagingbanner-banner-container-flex-direction);
+  margin: var(--amplify-components-inappmessagingbanner-banner-container-margin);
+  padding: var(--amplify-components-inappmessagingbanner-banner-container-padding);
+  position: var(--amplify-components-inappmessagingbanner-banner-container-position);
+  right: var(--amplify-components-inappmessagingbanner-banner-container-right);
+  row-gap: var(--amplify-components-inappmessagingbanner-banner-container-row-gap);
+}
+@media only screen and (min-width: 600px) {
+  .amplify-in-app-messaging-banner__container {
+    max-width: 50%;
```

Comment:
This will apply any time the screen width is greater than 800px.

Reply:
Hmm wouldn't it just stick with 400px even at screen widths greater than 800px? I don't think I saw a `width` defined anywhere that would cause this to ever try to grow beyond 400px